osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayer.cs

```diff
@@ -179,9 +209,6 @@ public void TestJoinRoomWithPassword()
             AddUntilStep("password prompt appeared", () => (passwordEntryPopover = InputManager.ChildrenOfType<DrawableRoom.PasswordEntryPopover>().FirstOrDefault()) != null);
             AddStep("enter password in text box", () => passwordEntryPopover.ChildrenOfType<TextBox>().First().Text = "password");
             AddStep("press join room button", () => passwordEntryPopover.ChildrenOfType<OsuButton>().First().Click());
-
-            AddUntilStep("wait for room open", () => this.ChildrenOfType<MultiplayerMatchSubScreen>().FirstOrDefault()?.IsLoaded == true);
-            AddUntilStep("wait for join", () => client.Room != null);
```

Comment:
Revert this. I want to make sure that this asserts the room was actually joined anyway.

Reply:
This was just a commit failure, not intended.